gtsfm/bundle/bundle_adjustment.py

```diff
@@ -36,7 +38,7 @@ class BundleAdjustmentOptimizer(NamedTuple):
 
     output_reproj_error_thresh: float
 
-    def run(self, initial_data: GtsfmData) -> GtsfmData:
+    def run(self, initial_data: GtsfmData) -> Tuple[GtsfmData, GtsfmMetricsGroup]:
```

Comment:
docstring for return type needs to be updated?

Reply:
i think we still need to update this

Reply:
i updated it